IOPool/Input/src/RootFile.cc

```diff
@@ -803,11 +801,11 @@ namespace edm {
     if (duplicateChecker_.get() == nullptr) {
       return false;
     }
-    fillEventAuxiliary(indexIntoFileIter_.entry());
+    auto const& eventAux = fillEventAuxiliary(indexIntoFileIter_.entry());
```

Comment:
No other call to `fillEventAuxiliary` take the return value to a (const) reference. It wouldn't make much difference in practice, but I wonder if this could be
```suggestion
    auto const eventAux = fillEventAuxiliary(indexIntoFileIter_.entry());
```
for consistency? (or should some other calls use const reference?)

Reply:
changed